scripts/firebase-deploy

```diff
@@ -69,6 +72,28 @@ get_pr_num() {
 	echo "${pr_num}"
 }
 
+# update_links_in_file updates links in the given HTML file to subdomain,
+# e.g.: href="/docs" → href="https://docs.evy.dev" relevant subdomains are:
+# discord, docs, learn and play.
+update_links_in_file() {
+	local domain="$1" file="$2" target
+	target="firebase/${file#out/}"
+	sed -E -e 's href="/(discord|docs|learn|play) href="https://\1.'"${domain} g" "${file}" >"${target}"
+	diff -u "${file}" "${target}"
```

Comment:
Is this leftover debugging? Or did you want to always show the diff?

Reply:
This is purposeful, I wanted to show it just in case cause the whole thing is a little hacky. Bad idea?

Reply:
No, all good. Just checking